control_panel_api/migrations/0001_initial.py

```diff
@@ -42,16 +47,146 @@ class Migration(migrations.Migration):
             ],
         ),
         migrations.CreateModel(
-            name='Project',
+            name='App',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('created', models.DateTimeField(auto_now_add=True)),
+                ('created', django_extensions.db.fields.CreationDateTimeField(auto_now_add=True, verbose_name='created')),
+                ('modified', django_extensions.db.fields.ModificationDateTimeField(auto_now=True, verbose_name='modified')),
                 ('name', models.CharField(max_length=100)),
-                ('slug', models.CharField(max_length=100)),
-                ('repository', models.CharField(blank=True, default='', max_length=255)),
+                ('slug', django_extensions.db.fields.AutoSlugField(blank=True, editable=False, populate_from='name')),
```

Comment:
I think [this is wrong now](https://github.com/ministryofjustice/analytics-platform-control-panel/blob/api/control_panel_api/models.py#L45), I don't know if it's because the squash happened before [that change](https://github.com/ministryofjustice/analytics-platform-control-panel/pull/36)?

This automated squashing is not giving me much confidence to be honest... :)